src/main/webapp/resources/js/contexts/AnalysisContext.js

```diff
@@ -44,6 +44,30 @@ const AnalysisContext = React.createContext(initialContext);
 function AnalysisProvider(props) {
   const [analysisContext, setAnalysisContext] = useState(initialContext);
 
+  // Update the analysis details that
+  // are required to display the progression
+  // using polling
+  useEffect(() => {
+      const interval = setInterval(() => {
+        getUpdatedDetails(analysisContext.analysis.identifier).then(res => {
+          setAnalysisContext(analysisContext => {
+            return {
+              ...analysisContext,
+              analysisState: res.analysisState,
+              isCompleted: res.analysisState === "COMPLETED",
+              isError: res.analysisState.includes("ERROR"),
+              previousState: res.previousState,
+              duration: res.duration
+            };
```

Comment:
You should catch an error thrown from the server.  Stop the polling and show a notice to the user that "something" went wrong.

Reply:
Updated in c64a856128d212fdda1bcab36dbf56b78cef8399